modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/modals/DeleteAPIApplicationModalContent.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayModal from '@clayui/modal';
+import classNames from 'classnames';
+import {fetch, openToast, sub} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+interface DeleteAPIApplicationModal {
+	closeModal: voidReturn;
+	itemData: ItemData;
+	loadData: voidReturn;
+}
+
+export function DeleteAPIApplicationModalContent({
+	closeModal,
+	itemData,
+	loadData,
+}: DeleteAPIApplicationModal) {
+	const [confirmed, setConfirmed] = useState(false);
+	const [displayError, setDisplayError] = useState(false);
+
+	async function handleDelete() {
+		const deleteURL = itemData.actions.delete.href;
+
+		fetch(deleteURL.replace('{id}', String(itemData.id)), {
+			method: 'DELETE',
+		})
+			.then(({ok}) => {
+				if (ok) {
+					closeModal();
+					loadData();
+					openToast({
+						message: Liferay.Language.get(
+							'your-request-completed-successfully'
+						),
+						type: 'success',
+					});
+				}
+				else {
+					throw new Error();
+				}
+			})
+			.catch(() => {
+				openToast({
+					message: Liferay.Language.get(
+						'an-unexpected-error-occurred'
+					),
+					type: 'danger',
+				});
+			});
+	}
+
+	const handleConfirmtionInput = (value: string) => {
```

Comment:
Shouldn't be `handleConfirmationInput?

Reply:
Yes, it should! Will fix it and resend it. Thanks